src/screens/UserPortal/Organizations/Organizations.spec.tsx

```
@@ -21,48 +22,118 @@ import { StaticMockLink } from 'utils/StaticMockLink';
+      event: React.MouseEvent<unknown> | null,
+      newPage: number,
+    ) => void;
+    onRowsPerPageChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  }) => (
+    <div data-testid="pagination">
+      <span data-testid="current-page">{page}</span>
+      <button
+        data-testid="prev-page"
+        onClick={(e) => onPageChange(e, page - 1)}
+        disabled={page === 0}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)